docs/content/docs/dev_guide/_index.md

```diff
@@ -10,9 +10,9 @@ weight: 5
 Welcome to the Open Service Mesh development guide!
 Thank you for joining us on a journey to build an SMI-native lightweight service mesh. The first of our [core principles](https://github.com/openservicemesh/osm#core-principles) is to create a system, which is "simple to understand and contribute to." We hope that you would find the source code easy to understand. If not - we invite you to help us fulfill this principle. There is no PR too small!
 
-To understand _what_ Open Service Mesh does - take it for a spin and kick the tires. Install it on your Kubernetes cluster by following [this guide](./example/README.md).
+To understand _what_ Open Service Mesh does - take it for a spin and kick the tires. Install it on your Kubernetes cluster by following [this guide](../install/).
```

Comment:
I believe this used to point to the manual demo guide earlier: https://github.com/openservicemesh/osm/blob/main/docs/content/docs/install/manual_demo/_index.md
```suggestion
To understand _what_ Open Service Mesh does - take it for a spin and kick the tires. Install it on your Kubernetes cluster by following [this guide](./install/manual_demo/_index.md).
```

Reply:
I'm not sure the relative link in my comment is correct, you may have to verify that it points to the above absolute link

Reply:
Hey @SanyaKochhar I think linking to main `/install` page seems better, in same page there is link to manual guide if needed.

Reply:
Hmm I agree that the sentence sounds like it should point to an install guide. The issue is that pointing to a directory instead of a markdown file is not going to render correctly on the [docs site](https://docs.openservicemesh.io/docs/dev_guide/). 

Maybe it would make sense to instead reword the sentence so it directs folks more clearly? Something like:
```suggestion
To understand _what_ Open Service Mesh does - take it for a spin and kick the tires with [this manual demo guide](../install/manual_demo/_index.md).
```
This manual demo guide itself points users/developers to the install guide, so I think that would be a good flow to flow to follow to figure out what OSM does. What do you think?